backend/accounts/baseaccount.go

```diff
@@ -289,6 +304,7 @@ func (account *BaseAccount) ExportCSV(w io.Writer, transactions []*TransactionDa
 			// a multitx is output in one row per receive address. Show the tx fee only in the
 			// first row.
 			feeString = ""
+			feeUnit = ""
```

Comment:
In receive transactions in Bitcoin, the feeString is `""`, but the unit "satoshi" is shown. But here, if the fee is `""`, the fee unit is not shown. Would be good to be consistent, i.e. show the feeUnit always, or never show it if the feeString is `""`.

Reply:
I went with the second: never showing it if the `feeString` is empty. LMK if this looks ok to you

Reply:
And I added some unit tests to make sure :P